Add multiple options

mhergon opened the pull request:
Added:
###### 

NS_ENUM(ValueLabelPositionType)
|--- ValueLabelLeftInside
|--- ValueLabelLeftOutside

@property (nonatomic) BOOL drawVerticalGrid;
@property (nonatomic) BOOL drawHorizontalGrid;

Removed:
###### 

NS_ENUM(ValueLabelPositionType)
|--- ValueLabelLeft

Please, search for "// @mhergon Pull request" to see my changes

mhergon committed:
Added:

NS_ENUM(ValueLabelPositionType)
|--- ValueLabelLeftInside
|--- ValueLabelLeftOutside

@property (nonatomic) BOOL drawVerticalGrid;
@property (nonatomic) BOOL drawHorizontalGrid;

Removed:
NS_ENUM(ValueLabelPositionType)
|--- ValueLabelLeft

Please, search for "// @mhergon Pull request" to see my changes

ArthurGuibert commented:
Thanks for the pull request! I'm going to have a look. I just had a quick look and there are a few things that need to change before I consider merging:
 - I'd rather not have the "// @mhergon Pull request" as I can see the diffs on github anyway.
 - I saw that you changed the example. I'm not sure I like it this way... would you mind revert it back as the way it is in the master branch?

mhergon commented:
Hi,

I'm sorry, you are right. I modified to test with my project and I forget delete these lines.
If you want, I send you the new and modified lines and you change on main branch.
Is that ok?

Thanks!
